packages/aws_elb_otel/manifest.yml

```
@@ -0,0 +1,34 @@
+format_version: 3.5.0
+name: aws_elb_otel
+title: "AWS ELB OpenTelemetry Assets"
+version: 0.1.0
+source:
+  license: "Elastic-2.0"
+description: "AWS ELB logs for OpenTelemetry Collector"
+type: content
+categories:
+  - aws
+  - cloud
+  - web
+  - observability
+  - opentelemetry
+conditions:
+  kibana:
+    version: "^9.2.0"
+  elastic:
+    subscription: "basic"
+discovery:
+  fields: []
```

```suggestion
discovery:
  fields: []
  datasets:
    - name: aws.elbaccess.otel
```

Please apply before merging - also need to make sure the data_stream name is correct as it might change yet

cc @ishleenk17 

See https://github.com/elastic/obs-integration-team/issues/623#issuecomment-3405449429